app/src/main/java/org/mozilla/fenix/home/sessioncontrol/SessionControlController.kt

```diff
@@ -46,16 +39,6 @@ import mozilla.components.feature.tab.collections.Tab as ComponentTab
  */
 @SuppressWarnings("TooManyFunctions")
 interface SessionControlController {
```

Comment:
todo: nit: lets rename this from sessionControlController to home something controller